### DISCUSSION TITLE
Popper's critique of the Big Bang theory

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#119] [Metaphysics and Philosophy]
<<Popper plunges into a technical critique of the Big Bang theory. “It's always the same,” he sums up. “The difficulties are underrated. It is presented in a spirit as if this all has scientific certainty, but scientific certainty doesn't exist.”>>
in "The Popper Paradox" by John Horgan
🔗[https://johnhorgan.org/cross-check/the-paradox-of-karl-popper](https://johnhorgan.org/cross-check/the-paradox-of-karl-popper)

**Comment by Pierre314159** [comment]
Not just « presented in a spirit »; it is squarely qualified as [« incontrovertible » and « fact »](https://www.quora.com/Do-you-believe-in-the-Big-Bang-theory-or-creation-and-why/answer/Stephen-Perrenod?ch=17&oid=31735724&share=5d2bb24a&srid=5Ahcp&target_type=answer).

If I had a PhD in physics, I would be ashamed to share a common title with Stephen Perrenod. Fortunately, i don’t.

**Reply by khuramonline** [reply]
It was this type of dogmatic "physicists" on quora who compelled me to write books against big bang and dark matter. I was just writing a book on Epistemological Realism and had joined quora only to ask questions about physics from experts -- only to clear up my mind about reality -- as my pursuit was realism.

I already had somewhat idea of the ongoing dogmatic state of Physics. But had no such idea that to what extent physics have turned into a cultic and dogmatic belief system.

Top physics experts on quora are a disgrace to science and physics. They behave just like cult follower puppets. And they are not the root of the problem. They are just branches. It is responsibility of actual intellectuals and scholars to pin point the actual roots of the problem.

One early root was Eddington. He just wrote a sentence in 1931 -- that under GR, universe is unstable. Either it has to expand or contract.

And that sentence is still vehemently followed by the so-called scientific community with full cultic strength.   

Anyways, my original project on Epistemological Realism is still pending.

**Comment by khuramonline** [comment]
Popper is right about Logical Positivists and Wittgenstein. Wittgenstein made a branch within logical positivism and they promoted the non-questioning attitude towards science. In first chapter of my book (2019), I wrote that Philosophy was dumped by Philosophers (Logical Positivists) and meanwhile another Rationalist Philosophy emerged under the guise of science by the name of Theoretical Physics.

Ground for non-critical attitude for science was ready. And Theoretical Physics i.e. a form of Philosophy took the benefit of that ready ground.

Wittgenstein's original aim was only to get himself understood by only one person. But since he got preface or forward for his book written by Bertrand Russel, a popular figure, so he unduly became popular. People started blindly following him. It was better for him if only one person actually understood him and then wrote a complete rebuttal of his each and every point.  

But due to having become popular, Wittgenstein turned into the greatest disgrace to philosophy.

**Reply by HanDeBruijn** [reply]
Popper: “_Most philosophers are really deeply depressed, because they can’t produce anything worthwhile. It would be better not to write that, I have enough enemies. But it is so_.”

**Reply by khuramonline** [reply]
Here Popper is explaining why he himself is not depressed.

Anyhow, I have a brief blog post on this topic. I quote below:

[Can there be role of Philosophy towards the Progress of Science?](https://bbtrejected.wordpress.com/2019/11/04/can-there-be-role-of-philosophy-towards-the-progress-of-science/)

> The answer to this question is “Yes and No”. More precise answer is “No and Yes”.
> 
> No part is because Philosophers tend to transcend physical reality. They may or may not go to right side. Even if some of them reach to right destination of transcendent reality, their method is not suitable to study physical reality.
> 
> For the yes part … (i) Philosophy must not pursue transcendent destination beyond of physical reality and (ii) Philosophy must be assertive of physical reality.

**Reply by khuramonline** [reply]
Since it is relevant, I also quote another of my blog posts:

[Philosophical Approach Vs. Contemporary ‘Scientific’ Approach](https://bbtrejected.wordpress.com/2019/05/12/philosophical-approach-vs-contemporary-scientific-approach/)

Present day science — specifically Theoretical Physics is based on body of knowledge and rigid frameworks already worked out by various scientists during the last hundred years. New scientist is required to take start from the point where previous scientists had reached. This system, by and large, works on trust that overall body of knowledge worked out by others is correct and a system of citations is used to build the body of knowledge layer upon layer.

On the other hand, Philosopher is one who must start from scratch. He must start from a point where he has removed all the doubtful elements. Starting point must be confirmed to his full satisfaction. Present day type Science can function correctly as long as it is already 100% correct. But many substandard and even fake ideas merge, submerge and grow within this science. There is a weak citation method where new scientist just trust works or results of others and starts building his own castles of ideas or theories. New scientists thus start building their extensions on an ill structured building. A stage comes when only solution left is that building may be bulldozed and re-constructed from scratch. Only a good philosopher can do that.

Anyhow, a major reformation in citation methodology is needed within scientific method that new scientists must not be allowed to merely cite the works of others. Whatever they are citing, they must analyze it and reach to the conclusion that they are citing it because they agree to it after thorough verification.  

If above mentioned major reformation is not introduced then Philosophers should come forward to bulldoze the ill structured body of knowledge with accurate criticism and start the things on solid foundations with the methodology to cite works of others only after thorough analysis and verification.

Ideal vs real science is a reality. People generally defend official science because they trust that only ideal things are going on. Actually there are lot of factual inaccuracies in various science papers – and science people keep citing those factual inaccuracies.

Following are examples:

1- Alaxender Friedmann (1922) ‘predicted’ Hubble Law.

2- Lemaitre (1927) ‘predicted’ Hubble Law out of GR equations.

3- Hubble (1929) found ‘Expanding Universe’.

4- GR equations originally had the ‘prediction’ of either expanding or contracting universe and that Einstein introduced Cosmological Constant only to stay with prevailing point of view of static universe.

List may go on … all above are factually wrong statements … but they have been merged, submerged and they keep growing under established scientific citation method.

Scientists only follow “standard format” of citation … they do not actually analyze and confirm the reality of what they are citing. A handsome number of citations are to the credit of Dunning and Krugger Effect and no one including Dunning and Krugger has ever bothered to confirm that case of Mc Auther Wheeler was already well theorized by Sir James Frazer and there was no need of a substandard theory of DK effect when a better theory was already available.

I have asked following question directly from Mr. Dunning … but no response … In fact this actually delivered system works this way … as opposed to advertisements … He will not respond.

[Why were Dunning and Kruger themselves not incompetent as they did not realize that the case of McAuther Wheeler related to the phenomenon of ‘Sympathetic Magic’ that was already well studied and explained by Sir James Frazer?](https://www.quora.com/Why-were-Dunning-and-Kruger-themselves-not-incompetent-as-they-did-not-realize-that-the-case-of-McAuther-Wheeler-related-to-the-phenomenon-of-Sympathetic-Magic-that-was-already-well-studied-and-explained-by-Sir)

Philosophy is not standardized … Yes now ‘Academic Philosophy’ is standardized … but they are only publish or perish type of people who are maintaining their job careers and they are the ones who have completely ruined the spirit of Philosophy. The actual spirit of philosophy is to start from scratch … or if cite anyone … then first analyze that one and conclude that one by yourself and then cite with your own conclusion.

Yes Philosophy is full of non-sense blunders. But it is like digging the earth. By ratio you will get greater amounts of useless soil and things … but gems will also be found in lesser ratio off course. Here sand and soil cannot be blamed as output of Philosophy as they are essential part of doing effort to reach at the level of gems.

And by the way, Theoretical Physics is actually a form of Rationalism Philosophy … but obviously science people are not going to accept it.

[Theoretical Physics is a form of Rationalism Philosophy:](http://bbtrejected.wordpress.com/2019/05/13/theoretical-physics-is-a-form-of-rationalism-philosophy/)

**Reply by HanDeBruijn** [reply]
@khuramonline .

>I also quote another of my blog posts:

Everybody can click on the link and read what you have written.
It's a waste of bandwith to simply repeat all of the content again at this place.

**Reply by khuramonline** [reply]
There are broken links in blog entries. When I share here I update the links here. In free time I will update all the links on blog.

**Comment by OlliSant** [comment]
That is fine to analyse the basic philosophy of BB theory. It is not scientific, it is pseudoscience in many ways. More of this in ACG! I hope.
